person.rb

```diff
@@ -0,0 +1,30 @@
+class Person < Nameable
+  attr_reader :id
+  attr_accessor :age, :name
+
+  def initialize(age, name = `unknown`, parent_permission: true)
+    super()
+    @id = DateTime.now
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
+
```

Comment:
- [ ] Kindly be noticed that the `Person` class should have a method `correct_name`.
```ruby
def correct_name
   name
end
```